addons/pkg-npm/jaxrs/src/main/java/org/commonjava/indy/pkg/npm/jaxrs/NPMContentAccessHandler.java

```diff
@@ -73,6 +73,8 @@
 
     public static final String TEMP_EXTENSION = ".temp";
 
+    private GeneratedTransfers generatedTransfers = new GeneratedTransfers();
```

Comment:
This is keeping state at the component level, and because the component is @ApplicationScoped it acts basically the same as a static variable. I'll add more information about this below.